pkg/lib/oauth/oidc/id_token_test.go

```diff
@@ -48,4 +59,104 @@ func TestSID(t *testing.T) {
 		_, _, ok = DecodeSID(EncodeSID(s))
 		So(ok, ShouldBeFalse)
 	})
+
+	Convey("IssueIDToken and VerifyIDTokenHint", t, func() {
+		ctrl := gomock.NewController(t)
+
+		now := time.Date(2020, 1, 1, 0, 0, 0, 0, time.UTC)
+
+		secrets := &config.OAuthKeyMaterials{
+			Set: jwk.NewSet(),
+		}
+		jwkKey := utilsecrets.GenerateRSAKey(now, rand.SecureRand)
```

Comment:
Instead of generating a 2048-bit RSA key pair per test, generate it once and paste it here in PEM format. You can find a example in jwkutil/pem_test.go You can just copy the `PrivateKeyPEM` in jwkutil/pem_test.go and use it here.